src/Raven.Client/Util/TcpUtils.cs

```diff
@@ -204,8 +204,11 @@ Logger log
             if (info.Certificate == null)
                 return networkStream;
 
-            var expectedCert = new X509Certificate2(Convert.FromBase64String(info.Certificate), (string)null, X509KeyStorageFlags.MachineKeySet);
-            var sslStream = new SslStream(networkStream, false, (sender, actualCert, chain, errors) => expectedCert.Equals(actualCert));
+            var sslStream = new SslStream(networkStream, false, (sender, actualCert, chain, errors) =>
+            {
+                string actualStringCert = Convert.ToBase64String(actualCert.Export(X509ContentType.Cert));
```

Comment:
previously we compared 2 instances of X509Certificate2, now we are comparing 2 Base64 strings. Is this what we want? Want to double check that. In all cases 'actualCert' is exportable?

Reply:
Done